<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1578</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_west_gate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>1.西门</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_song_restaurant">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>2.松苑餐厅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mei_restaurant">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>3.梅苑餐厅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tao_restaurant">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>4.桃苑餐厅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_teching_building">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>6.博学楼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_library">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>5.图书馆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_deer_park">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>7.鹿苑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_academy_building">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>8.学院楼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_confucius_statue">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>9.夫子像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_east_gate">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>280</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>10.东门</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_min_path">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>290</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>最短的路</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_start_point">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>180</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_end_point">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>220</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result_min_path">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>180</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelstartpoint">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>起点序号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_end_point">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>230</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>终点序号</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1578</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
